src/css/shepherd-custom.css

```diff
@@ -78,3 +74,79 @@
     bottom: 5px;
     right: 5px;
 }
+
+.guide-dialog.shepherd-element.menu-setup-guide-dialog {
+    margin-top: 12px;
+    margin-left: 5px;
+}
+
+.guide-dialog.shepherd-element.sub-menu-repositories-guide-dialog,
+.guide-dialog.shepherd-element.sub-menu-autocomplete-guide-dialog,
+.guide-dialog.shepherd-element.sub-menu-visual-graph-guide-dialog,
+.guide-dialog.shepherd-element.sub-menu-class-hierarchy-guide-dialog
+{
+    margin-left: 12px;
+}
+
+.guide-dialog.shepherd-element.import-file-button-guide-dialog,
+.guide-dialog.shepherd-element.visual-graph-side-panel-content-guide-dialog,
+.guide-dialog.shepherd-element.visual-graph-properties-focus-guide-dialog,
+.guide-dialog.shepherd-element.visual-graph-properties-side-panel-close-guide-dialog,
+.guide-dialog.shepherd-element.clas-hierarchy-intro-guide-dialog,
+.guide-dialog.shepherd-element.class-hierarchy-zoom-content-guide-dialog,
+.guide-dialog.shepherd-element.class-hierarchy-instances-guide-dialog,
+.guide-dialog.shepherd-element.class-hierarchy-side-panel-info-guide-dialog,
+.guide-dialog.shepherd-element.class-hierarchy-side-panel-close-guide-dialog
+{
+    margin-left: -12px;
```

Comment:
Are these working properly on different screen resolutions?

Reply:
I tested it on some resolutions, and it looks fine. However, I can't guarantee that everything will appear okay on all resolutions, especially if the resolutions are too small or in the visual graph where nodes move after being displayed.